packages/calcite-components/src/components/stepper-item/stepper-item.scss

```diff
@@ -17,9 +17,9 @@
  * @prop --calcite-stepper-bar-fill-color: Specifies the component's fill color.
  * @prop --calcite-stepper-bar-fill-color-hover: Specifies the component's fill color when hovered or focused.
  * @prop --calcite-stepper-bar-complete-fill-color: Specifies the component's fill color when complete.
- * @prop --calcite-stepper-bar-complete-fill-color-hover: When item is complete, specifies the component's fill color when hovered or focused.
- * @prop --calcite-stepper-bar-error-fill-color: Specifies the component's fill color when in error.
- * @prop --calcite-stepper-bar-error-fill-color-hover: When item is error, specifies the component's fill color when hovered or focused.
+ * @prop --calcite-stepper-bar-complete-fill-color-hover: Specifies the component's fill color when hovered or focused and is complete.
+ * @prop --calcite-stepper-bar-error-fill-color: Specifies the component's fill color when in an error state.
+ * @prop --calcite-stepper-bar-error-fill-color-hover: Specifies the component’s fill color when hovered or focused and in an error state.
```

Comment:
Similar to above, the doc consistency seems okay here as well. Maybe some context to update what "item" refers to here instead?

Something like:
```
When the component is `complete`, specifies the fill color when hovered or focused.
...
When the component contains an `error`, specifies the component's fill color when hovered or focused.
```